internal/jobobject/jobobject.go

```diff
@@ -433,7 +442,8 @@ func (job *JobObject) QueryProcessorStats() (*winapi.JOBOBJECT_BASIC_ACCOUNTING_
 	return &info, nil
 }
 
-// QueryStorageStats gets the storage (I/O) stats for the job object.
+// QueryStorageStats gets the storage (I/O) stats for the job object. job.SetIOTracking() must be
+// invoked or when the job was created `EnableIOTracking` must be set to true for this call to function.
```

Comment:
The "job.SetIOTracking() must be invoked" runs into the "or when the job was created `EnableIOTracking` must be set to true" bit, so its hard to parse what is being said

Reply:
Reworded